frontend/src/pages/friendDashboard.tsx

```
@@ -1,99 +1,132 @@
+      navItems={mainNavItems}
+      headerRightContent={
+        <button
+          onClick={handleAddFriend}
+          disabled={isAlreadyFriend}
+          className={`inline-flex items-center px-4 py-2 rounded-lg
+          transition-all duration-200 shadow-sm hover:shadow
+          font-medium text-sm gap-2
+          ${isAlreadyFriend 
+            ? 'bg-gray-100 text-gray-600 cursor-default' 
+            : 'bg-green-600 text-white hover:bg-green-700'}`}
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)